Sources/SnapshotTesting/Strategy/Codable.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-extension Strategy where A: Encodable {
+extension Strategy where A: Encodable, B == String {
```

Comment:
@stephencelis ok to further constraint to string?

Reply:
Yeah we should probably reconstrain everything. Makes the pullback more annoyingly verbose, but namespaces better.